### PR TITLE
template: Refactor service tmpl

### PR DESCRIPTION
**Pull request by Xuanwo**
Signed-off-by: Xuanwo <xuanwo@yunify.com>

### CODE REVIEW

File: src/QingStor/QingStor.php [hunk 1]
```diff
@@ -102,6 +119,10 @@ public function listBucketsQuery($expires, $options = array())
         return $signer->query_sign($expires);
     }
 
+    public function listsValidate($operation)
```
Comment (prettyxw) [inline src/QingStor/QingStor.php]: Maybe listBucketsValidate()?

File: template/service.tmpl [hunk 2]
```diff
@@ -36,117 +37,9 @@ class {{$service.Name | camelCase}}Service
...
-     * @return Signer
-     */
-    public function {{$operation.ID | camelCase | lowerFirst}}Query($expires {{- if $hasInput}}, $options=array(){{end -}})
-    {
-        $signer = $this->{{$operation.ID | camelCase | lowerFirst}}Request({{- if $hasInput}}$options{{end -}});
-        return $signer->query_sign($expires);
-    }
-    {{end}}
+    {{- range $_, $operation := $service.Operations}}
+    {{- template "RenderOperation" passThrough $service.Name $operation $customizedTypes}}
```
Comment (prettyxw) [inline template/service.tmpl]: Missing indent.

File: template/shared.tmpl [hunk 3]
```diff
@@ -1,14 +1,14 @@
 {{define "SubServiceInitParams"}}
-    {{- $customizedType := . -}}
-    {{- $firstPropertyID := $customizedType | firstPropertyIDInCustomizedType -}}
-    {{- range $_, $property := $customizedType.Properties -}}
-        {{- if ne $property.ID $firstPropertyID -}},{{- end -}}
+    {{- $customizedType := . }}
```
Comment (prettyxw) [inline template/shared.tmpl]: Remove space before the right curly bractets.

**Comment by Xuanwo** [comment]
@prettyxw All fixed, please review again.

**Pull request merged**